static/styles/settings.css

```diff
@@ -1408,6 +1408,7 @@ $option_title_width: 180px;
 #edit-user-form {
     .custom_user_field {
         padding-bottom: 20px;
+        width: fit-content;
```

Comment:
This should instead be set on the `div.field` element so that the tooltip is not shown outside the `Pronouns` type field input.